dashboard/app/models/regional_partners_school_district.rb

```diff
@@ -4,6 +4,8 @@
 #
 #  regional_partner_id :integer          not null, primary key
 #  school_district_id  :integer          not null, primary key
+#  course              :string(255)
```

Comment:
What are the possible choices for choice? Should this model include some validation on this field? Etc.